README.md

```diff
@@ -107,6 +107,7 @@ Options include:
 (e.g. `--scope read_products, write_orders, write_products` or `--scope "read_products, write_orders, write_products"`)
 For more information, refer to the [docs](http://docs.shopify.com/api/tutorials/oauth).
 * `embedded` - the default is to generate an [embedded app](http://docs.shopify.com/embedded-app-sdk), if you want a legacy non-embedded app then set this to false, `--embedded false`
+* __[Not recommended]__ `with-cookie-authentication` - sets up the authentication strategy of the app to use cookies. By default, it uses JWT based session tokens.
```

Comment:
Even though we don't want cookies for embedded apps, we still need them for non-embedded ones right? Is `not recommended` the right wording here?

Reply:
We would still need them for non-embedded apps. This is a good point. By default, non-embedded apps would be setup to use JWTs, which will not work for them. Would removing `not recommended` be sufficient in supporting them?

Reply:
Is there a way to prompt the user for their option when running the generator? That would enable us to set things up to match what they're going for.

Reply:
@paulomarg we could do that. Although it might make things a bit more confusing now that the generator branches off for embedded and non-embedded apps. Do you suggest we add an option specifying what app is being generated?

Reply:
Looks like there's already [an `embedded` option](https://github.com/Shopify/shopify_app/pull/1111/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R109). Maybe that can be leveraged to ensure the right path is taken based on the app choice. _(Although the documentation link points to EASDK and I'm assuming that's an outdated link)._

And the doc wording here can accordingly be updated to `[Not recommended for embedded apps]`

Reply:
I think having the user tell us explicitly whether they're creating an embedded or non-embedded app may be helpful. I agree with Sneha that the `embedded` flag seems like a good fit for this (depending on what else it might do).

Reply:
Since `embedded` is the default configuration, I can make sure that the `non-embedded` option uses cookies.

Reply:
So I take it the new strategy is to drop the "with-cookie-authentication" flag and leverage the "embedded" flag? That makes sense to me. 👍